README.md

```diff
@@ -196,6 +196,21 @@ When experimenting with kinds, keep in mind the classification introduced by [NI
 4. There should be no more than one way of doing the same thing.
 5. Other rules will be made up when necessary.
 
+## Mailing Lists
+
+The nostr ecosystem is getting large with many different organizations, relays
+and clients. Following the nips repo on github is becoming more difficult and
+noisy. To coordinate on protocol development outside of github, there are
+mailing lists where you can work on NIPs before submitting them here:
+
+* [w3c nostr community group][w3-nostr] - [public-nostr@w3.org][mailto-w3] - requires signup
+* [nostr-protocol google group][nostr-google-group] - [nostr-protocol@googlegroups.com][mailto-google] - no signup required
```

Comment:
I think a mailing list hosted by the Linux Foundation could be considered as documented here like we’re doing for bitcoin dev and lightning dev ones: https://lists.linuxfoundation.org/mailman/create

Linux foundation is a non-profit organization with a dedicated pledge of being a neutral platform for the world of open-source with 20 years of active history and presence on many continents.

Google is google :)

Overall it’s a much necessary thing, thanks for starting the maintenance.